casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -16,9 +16,14 @@ object DeployRuntime {
   def propose[F[_]: DeployService: Monad](): F[Unit] =
     DeployService[F].createBlock().flatMap {
       case Some(block) =>
-        DeployService[F].addBlock(block)
+        println(s"Response: Successfully created block")
```

Comment:
println leftover

Reply:
I didn't want to change how the existing communication is done (via println's).
WDYT - would it be better to introduce a logger?
The goal is (AFAI understand it) to communicate with the user via simple messages.

Reply:
Oh, I saw usage of `Log` earlier but it was from node. This is probably a question to @rabbitonweb and @sebtomba

Reply:
The `Log` instance we have is server-side, but we want this information to be shown client-side, so I think `println` is the right call.